packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
+        "request": {
+          "__arrayItem": {
+            "endpointId": {
+              "__regexp": "^0x[a-fA-F0-9]{64}$"
+            },
+            "oisTitle": {},
+            "endpointName": {}
+          }
+        }
+      }
+    },
+    "nodeSettings": {
+      "providerIdShort": {
+        "__regexp": "^[a-fA-F0-9]{7}$"
+      },
+      "nodeVersion": {
+        "__regexp": "^([0-9]*\\.){2}([0-9]*)$"
+      },
+      "cloudProvider": {
+        "__regexp": "^aws$"
+      },
+      "region": {},
+      "stage": {},
+      "logFormat": {
+        "__regexp": "^(json|plain)$"
+      },
+      "chains": {
+        "__arrayItem": {
+          "id": {},
+          "type": {
+            "__regexp": "^evm$"
+          },
+          "providers": {
+            "__arrayItem": {
+              "name": {},
+              "url": {
+                "__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$"
+              },
+              "__optional": {
+                "blockHistoryLimit": {},
+                "minConfirmations": {}
+              }
+            }
+          },
+          "__optional": {
+            "providerAdminForRecordCreation": {
+              "__regexp": "^0x[a-fA-F0-9]{40}$"
+            },
+            "contracts": {
+              "__keyRegexp": "^(Airnode|Convenience)$",
+              "__objectItem": {
+                "__regexp": "^0x[a-fA-F0-9]{40}$"
+              }
+            }
+          }
+        }
+      }
+    },
+    "id": {
+      "__conditions": [
+        {
+          "__if": {
+            "__this": ".*"
+          },
+          "__rootThen": {
+            "security": {
+              "id": {
+                "__regexp": "^__match$",
+                "__level": "error"
+              }
+            }
+          }
+        }
+      ]
+    }
+  },
+  "security": {
+    "__any": {
```

Comment:
I'm not sure how `__any` works without a `then` statement. I think this would imply the README should be updated to include an example of this functionality: https://github.com/api3dao/airnode/tree/master/packages/validator#any

Reply:
Added to https://github.com/api3dao/airnode/issues/203
But in short it works exactly the same as in `__then`, if at least one nested object matches the structure specified by `__any`, it is valid specification, otherwise it will print an error.